sst/nights/facilities_temperatures_reports.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>top of the plot --&gt; right of the plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I removed this part. The telemetry from MTMount showed to be unliable. I noticed that the temperature reported by MTMount could change by a degree after a state transition. On top of it, it was way too stable for our analysis. </p>